src/components/pages/Offers/Offer/Stocks/__specs__/Stocks.spec.jsx

```diff
@@ -2057,6 +2083,39 @@ describe('stocks page', () => {
 
         pcapi.loadOffer.mockResolvedValue(noStockOffer)
       })
+      it('should allow the user to add activation codes option', async () => {
```

Comment:
```suggestion
      it('should allow the user to add activation codes option when offer is digital', async () => {
```

Sauf si, comme pour l'édition, tu souhaites plutôt mettre dans un describe `digital offer`
À ce moment là il faudrait juste garder le test d'en-dessous sans le `when not digital` et le mettre également dans le describe des événements
(same same pour l'édition)